pages/_app.js

```diff
@@ -67,18 +67,23 @@ export default class HomePage extends App {
               </Col>
               <Col medium={8} small={4} xsmall={2} large={8}>
                 <nav
+                  aria-label="Menu de Navegação"
```

Comment:
```suggestion
                  aria-label="Menu Principal"
```
Defining an aria-label with the name "Navigation" in a nav tag is a bit redundant. I suggest changing the name